contracts/iotube/TransferValidatorV3.sol

```diff
@@ -0,0 +1,135 @@
+pragma solidity <6.0 >=0.4.24;
```

Comment:
is this based on contracts/iotube/TransferValidator.sol? Why not contracts/iotube/TransferValidatorV2.sol instead?